### DISCUSSION TITLE
CTRL-LeftShift or CTRL-RightShift - not changing the UI to left or right alignment in a multilingual scenario

### DISCUSSION THREAD

**Discussion by ips972** [open-webui/open-webui#7305] [Issues]
Recent Update Causes Display Issues with Open WebUI
Since upgrading to version 0.4.x, I've encountered a peculiar problem with the Open WebUI interface. The issue primarily affects text alignment and display, especially in multilingual environments.

When using the Ctrl-LeftShift or Ctrl-RightShift keys to switch between languages, the UI fails to adjust properly. Instead of changing the text alignment to left or right as expected, English text is displayed with an odd rightward shift, almost as if it's being pushed from the left. Additionally, spaces seem to be inserted to the left of the text until a new word is typed.

This problem makes it challenging to work with multiple languages simultaneously, which is often necessary in today's globalized world. The current workaround involves changing the entire UI language in the settings, which can be tedious and time-consuming.

Hopefully, the development team will address this issue soon, providing a seamless multilingual experience once again.

**Comment by tjbck** [comment]
PR Welcome!

**Reply by ips972** [reply]
can't PR, no programming skills :-( 
only installation and integration ones.  :-)

**Reply by ips972** [reply]
still having problems with this. any ideas on how to bring back the UI's support for changing from left to right and right to left with ALT+SHIFT or CTRL+SHIFT??

**Comment by ips972** [comment]
any update on this? a real issue for us. thanks!
